webtau-rest-groovy/src/main/groovy/com/twosigma/webtau/http/HttpExtensions.groovy

```diff
@@ -63,10 +71,18 @@ class HttpExtensions {
         return http.put(url, new JsonRequestBody(requestBody), closureToHttpResponseValidator(validation))
     }
 
+    static def put(Http http, String url, Closure validation) {
```

Comment:
Not sure put with empty body really makes sense?  Post is often used with an empty body for action like resources, things which aren't CRUD but not sure put really is.

Reply:
Good point. I will leave for now since someone can just not use semantic REST.